main/zoe/api/zoe-contract-facet.md

```diff
@@ -36,8 +36,8 @@ Create an empty offer for recordkeeping purposes (Autoswap uses this to create t
 let poolOfferHandle = zoe.escrowEmptyOffer();
 ```
 
-## zcf.escrowOffer(payoutRules, offerPayments)
-- `payoutRules` `{Array <PayoutRulesElem>}`
+## zcf.escrowOffer(offerRules, offerPayments)
+- `offerRules` `{OfferRules}`
 - `offerPayments` `{Array <Payment>}`
 - Returns: `{OfferHandle}`
 
```

Comment:
Sorry, this was a change that only happened in the past few days when Part 6 of the stacked PRs finally got merged into Zoe 0.2.0. We replaced `zcf.escrowOffer` and `zcf.escrowEmptyOffer` with `zcf.getZoeService` which gives the contract the client/user facing Zoe to make offers if the contract needs to.